src/ui/common/src/components/pages/metric/id/index.tsx

```diff
@@ -237,7 +237,7 @@ const MetricDetailsPage: React.FC<MetricDetailsPageProps> = ({
         } else {
           // Render inline if possible
           return (
-            <Box>
+            <Box key={artifactId}>
```

Comment:
Linter complained: `Missing "key" prop for element in iterator`

Reply:
Yeah, totally expected if we're going to be iterating. As long as the artifactId is uuid or something unique, we're good to go.